build_tools/azure/ubuntu_atlas_lock.txt

```diff
@@ -6,21 +6,21 @@
 #
 cython==0.29.33
     # via -r build_tools/azure/ubuntu_atlas_requirements.txt
-exceptiongroup==1.1.3
+exceptiongroup==1.2.0
     # via pytest
 execnet==2.0.2
     # via pytest-xdist
 iniconfig==2.0.0
     # via pytest
-joblib==1.1.1
+joblib==1.2.0
```

Comment:
I assume that the diff would be smaller if we sync the `callback` branch with `main`

Reply:
yep, done